api/types/client.go

```diff
@@ -294,7 +294,76 @@ type GatewayInspectResponse struct {
 	Listeners  map[string]GatewayEndpoint
 }
 
+type VanStatusInfo struct {
+	Addresses  []AddressInfo    `json:"addresses"`
+	SiteStatus []SiteStatusInfo `json:"siteStatus"`
+}
+
+type AddressInfo struct {
+	RecType        string `json:"recType,omitempty"`
+	Identity       string `json:"identity,omitempty"`
+	Name           string `json:"name,omitempty"`
+	StartTime      uint64 `json:"startTime"`
+	EndTime        uint64 `json:"endTime"`
+	Protocol       string `json:"protocol,omitempty"`
+	ListenerCount  int    `json:"listenerCount"`
+	ConnectorCount int    `json:"connectorCount"`
+}
+
+type SiteStatusInfo struct {
+	Site         SiteInfo           `json:"site"`
+	RouterStatus []RouterStatusInfo `json:"routerStatus"`
+}
+
 type SiteInfo struct {
+	Identity       string `json:"identity,omitempty"`
+	Name           string `json:"name,omitempty"`
+	Namespace      string `json:"namespace,omitempty"`
+	Platform       string `json:"platform,omitempty"`
+	Version        string `json:"siteVersion,omitempty"`
+	MinimumVersion string `json:"minimumVersion,omitempty"`
+	Policy         string `json:"policy,omitempty"`
+}
+
+type RouterStatusInfo struct {
+	Router     RouterInfo      `json:"router"`
+	Links      []LinkInfo      `json:"links"`
+	Listeners  []ListenerInfo  `json:"listeners"`
+	Connectors []ConnectorInfo `json:"connectors"`
+}
+
+type RouterInfo struct {
+	Name         string `json:"name,omitempty"`
+	Namespace    string `json:"namespace,omitempty"`
+	Mode         string `json:"mode,omitempty"`
+	ImageName    string `json:"imageName,omitempty"`
+	ImageVersion string `json:"imageVersion,omitempty"`
+	Hostname     string `json:"hostname,omitempty"`
+}
+
+type LinkInfo struct {
+	Mode      string `json:"mode,omitempty"`
+	Name      string `json:"name,omitempty"`
+	LinkCost  uint64 `json:"linkCost,omitempty"`
+	Direction string `json:"direction,omitempty"`
+}
+
+type ListenerInfo struct {
+	Name     string `json:"name,omitempty"`
+	DestHost string `json:"destHost,omitempty"`
+	DestPort string `json:"destPort,omitempty"`
+	Protocol string `json:"protocol,omitempty"`
+	Address  string `json:"address,omitempty"`
+}
+
+type ConnectorInfo struct {
+	DestPort string `json:"destPort,omitempty"`
+	Address  string `json:"address,omitempty"`
+	Process  string `json:"process,omitempty"`
+	Target   string `json:"target,omitempty"`
+}
```

Comment:
Not a big issue, but to me it might be nice to have these new types in a different package. Perhaps in pkg/network for example?

Reply:
Yes, absolutely. Thanks!

Reply:
Solved in 68f66b2e5c69e3a9286832f7280c2938ebd36571